pkg/config/config_template.yaml

```diff
@@ -2858,10 +2858,11 @@ api_key:
   #
   # listen_address: /var/vcap/data/garden/garden.sock
 
-## @param podman_db_path - string - optional - default: /var/lib/containers/storage/libpod/bolt_state.db
+## @param podman_db_path - string - optional - default: ""
+## @env DD_PODMAN_DB_PATH - string - optional - default: ""
 ## Settings for Podman DB that Datadog Agent collects container metrics.
 #
-# podman_db_path: /var/lib/containers/storage/libpod/bolt_state.db
+# podman_db_path: ""
```

Comment:
Wouldn't that break anyone assuming the agent will use this path ?

Reply:
No thanks to https://github.com/DataDog/datadog-agent/pull/24373/files#diff-6e3b10722940c0034fff4b365221b7e011c34411ce1cef19ebff5fdaa4601c87R77-R89, tl;dr : with `podman_db_path` empty, if the previous value `/var/lib/containers/storage/libpod/bolt_state.db` is accessible, it will be used.